frontend/src/components/Layout/Header/HeaderContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useState } from 'react';
+
+import { User } from 'types/user';
+
+import Header from './Header';
+
+const HeaderContainer: React.FC<User> = ({ firstName, lastName, avatar }) => {
+  const [anchorElNotifications, setAnchorElNotifications] = useState<null | HTMLElement>(null);
```

Comment:
Rename `anchorElNotifications` to something more clear please
And maybe it will be better to store` boolean or null` value from the start in store, instead of adding new variables to the code?

Reply:
Renamed anchorElNotifications